README.md

```diff
@@ -72,15 +79,73 @@ After setting up your environment, you can verify your setup by sending a test m
   - **S3 URL**: Replace the S3 URL in the `--message` argument with the URL of the image file you want to test.
 
 2. Execute the following command, substituting your actual values:
-
     ```bash
-    python3 bin/local_cli.py --topic-arn <YOUR_TOPIC_ARN> --s3-uri <YOUR_S3_URI>
+    python3 bin/intake_cli.py --topic-arn <YOUR_TOPIC_ARN> --s3-uri <YOUR_S3_URI>
     ```
 
-3. **Expected Output**:
+3. Validate Expected Output:
   - This will trigger the processing of the specified image file in your application.
   - Verify that the auxiliary files are generated and uploaded to your configured S3 bucket, and ensure that the logs indicate a successful run.
 
+4. To put a test item directly in your STAC catalog, update the following command and run it with your endpoint:
```

Comment:
Just some suggestions, not really a blocker, but it would be nice to split into multiple sections here. For instance:

```
## Testing Locally

### Perform Ingest

<description and steps of ingest process>

### Perform Data Catalog 

<description and steps of how to execute curl commands etc etc>

### and so on, etc
```